ConsoleApp/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace ConsoleApp
+{
+    public class Program
+    {
+        static void Main()
+        {
+            Stopwatch timeBubbleSort = new Stopwatch();
+            Stopwatch timeQuickSort = new Stopwatch();
+
+            //Generate BIG array on 100 000 elements
+            int[] arr = GenerateArray(100000);
+
+
+            //Measure the execution time of the Bubble Sort
+            timeBubbleSort.Start();
+            int[] arrBubbleSort = BubbleSort(arr);
+            timeBubbleSort.Stop();
+
+            //Measure the execution time of the Quick Sort
+            timeQuickSort.Start();
+            int[] arrQuickSort = QuickSort(arr, 0, arr.Length - 1);
+            timeQuickSort.Stop();
+
+
+            //I dont want to print so big array)
+            //PrintArray(BubbleSort(arr));
+            Console.WriteLine($"Bubble sort time {arr.Length} elements: {timeBubbleSort.ElapsedMilliseconds} ms");
+
+            //PrintArray(arrQuickSort);
+            Console.WriteLine($"Quick sort time {arr.Length} elements: {timeQuickSort.ElapsedMilliseconds} ms");
+        }
+
+        //Generate array with n-elements
+        static int[] GenerateArray(int n)
+        {
+            Random random = new Random();
+            int[] arr = new int[n];
+
+            for (int i = 0; i < n; i++)
+            {
+                arr[i] = random.Next(-100, 101);
+            }
+            return arr;
+        }
+
+        //Bubble Sort Method
+        static int[] BubbleSort(int[] arr)
+        {
+            int[] copyArr = arr;
+
+            for (int i = 0; i < copyArr.Length; i++)
+            {
+                for (int j = 0; j < copyArr.Length - 1; j++)
```

Comment:
you do not need to go through the whole array each iteration, you can stop on **copyArr.Length - i**